pkg/cmd/server/start.go

```diff
@@ -34,15 +34,18 @@ import (
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apiserver/pkg/admission"
 	"k8s.io/apiserver/pkg/endpoints/openapi"
-	//"k8s.io/apiserver/pkg/features"
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	genericoptions "k8s.io/apiserver/pkg/server/options"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/klog/v2"
 	netutils "k8s.io/utils/net"
 )
 
-const defaultEtcdPathPrefix = "/registry/porch.kpt.dev"
+const (
+	defaultEtcdPathPrefix = "/registry/porch.kpt.dev"
+	OpenAPITitle   = "Porch"
+	OpenAPIVersion = "0.1"
```

Comment:
And here, could you explain what's going on?

Reply:
Just defining some constants for the openapi(v3) config.